### PR TITLE
[bellbot] fixes, locale support and tidying up

### PR DESCRIPTION
**Pull request by gatsoulis**
- bellbot_gui no longer writes on deb install folders. It uses `.ros/bellbot` and the `<deployment>_bellbot dirs` to do its write operations.
- Support for locale provided by `bellbot_gui` templates customized by `<deployment>_bellbot` configuration files
- Tidy up bellbot_action_server
- Commented out bellbot's two other modes of operations leaving only `WaitingForGoal`

### CODE REVIEW

**Comment by cdondrup** [comment]
Hi Yannis,

It would be great if you could not name PRs like tags and version numbers. This doesn't mean anything to me and there is no guarantee that this PR will be in version 0.0.11. Please give it speaking titles and also a description that may be a bit more helpful than "Various changes" ;) A list of short bullet points of what you did would be much appreciated.

@Jailander can you please have a look at this?

File: bellbot_action_server/package.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_action_server</name>
-  <version>0.0.10</version>
+  <version>0.0.11</version>
```
Comment (cdondrup) [inline bellbot_action_server/package.xml]: Never change version numbers manually! This will not be releasable now. The version number is changed by Jenkins automatically and ALL packages in this repository have to have the same version number.

File: bellbot_gui/package.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_gui</name>
-  <version>0.0.10</version>
+  <version>0.0.11</version>
```
Comment (cdondrup) [inline bellbot_gui/package.xml]: See comment for action server package.xml. This has to be changed to `0.0.10` again.

Comment (gatsoulis) [inline bellbot_gui/package.xml]: A ok, I thought you meant by PR that I had to do this... Didn't know these are changed by jenkins... How to fix that? Shall I manually change it back to 0.0.10 and commit?

File: bellbot_gui/scripts/gui2.py [hunk 3]
```diff
@@ -37,6 +36,7 @@ def florence():
                 rospy.logerr("virtual keyboard not found, please install it by running:\nsudo apt-get install florence")
     except:
         try:
+            pass
```
Comment (cdondrup) [inline bellbot_gui/scripts/gui2.py]: This doesn't do anything. Why is this necessary?

File: bellbot_action_server/package.xml (hunk 1 above)
Comment (gatsoulis) [inline bellbot_action_server/package.xml]: Didn't know that these are automatically updated by jen. How to fix it? Shall I manually change them back and commit?

File: bellbot_gui/package.xml (hunk 2 above)
Comment (cdondrup) [inline bellbot_gui/package.xml]: Yes, please change them back manually again.

File: bellbot_gui/scripts/gui2.py (hunk 3 above)
Comment (gatsoulis) [inline bellbot_gui/scripts/gui2.py]: There is another line below `pass` that is run... Anyway agree cause confusion so I can comment it out.

File: bellbot_gui/package.xml (hunk 2 above)
Comment (gatsoulis) [inline bellbot_gui/package.xml]: On 14/04/15 11:53, Christian Dondrup wrote:

> Yes, please change them back manually again.
> OK, done.

Regards,
Yiannis

**Comment by cdondrup** [comment]
Merging this to apply changes from the robot afterwards.
